packages/shell-api/src/replica-set.spec.ts

```diff
@@ -946,6 +946,53 @@ describe('ReplicaSet', () => {
         expect(status.members.length).to.equal(3);
       });
     });
+
+    describe('analyzeShardKey()', () => {
```

Comment:
Is this the right place to put the integration tests that makes sure these work without sharding just on a replicaset?